syntax = "proto3";

package mlpiot.proto;

import "mlpiot/proto/google_timestamp.proto";
import "mlpiot/proto/event.proto";


message EventExtractorMetadata {
    string name = 1;
    int32 version = 2;
    string payload = 3;
}

message EventExtraction {
    Timestamp timestamp = 1;
    EventExtractorMetadata metadata = 2;
    repeated Event events = 3;
}
